schema/google/showcase/v1beta1/compliance.proto

```diff
@@ -116,10 +119,13 @@ message RepeatRequest {
   optional int32 p_int32 = 7;
   optional int64 p_int64 = 8;
   optional double p_double = 9;
+
+  optional string intended_binding_uri = 10;
```

Comment:
I would place this near the `server_verify` field, since it's not testing the data fromat preservation like the rest of the compliance suite is

Reply:
:+1: